communication/moderators.md

```diff
@@ -7,18 +7,29 @@ See our [moderation guidelines](./moderating.md) for policies and recommendation
 
 ### kubernetes-dev
 
-### Administrators
+#### Administrators (3)
 
 - Sarah Novotny (@sarahnovotny) - PT
 - Brian Grant (@bgrant0607) - PT
+- Open
 
-### Moderators
+#### Primary Moderators (7)
 
 - Paris Pittman (@parispittman) - PT
 - Jorge Castro (@castrojo) - ET
 - Jaice Singer DuMars - (@jdumars) - PT
 - Louis Taylor (@kragniz)- CET
 - Nikhita Raghunath (@nikhita) - IT
```

Comment:
While we are here, can we change my timezone s/IT/IST? There is no timezone called IT. :)

IST = Indian Standard Time